resources/default_pipeline_environment.yml

```diff
@@ -493,7 +493,7 @@ steps:
       dockerName: 'bundler'
       dockerWorkspace: ''
   slackSendNotification:
-    color: "${buildStatus == 'SUCCESS'?'#008000':'#E60000'}"
+    color: "${['SUCCESS': '#8cc04f', 'FAILURE': '#d54c53', 'ABORTED': '#949393', 'UNSTABLE': '#f6b44b'].get(buildStatus)}"
```

Comment:
`NOT_BUILT` should also be in here.

https://github.com/jenkinsci/jenkins/blob/8427b7d1a2c67d303536166861b7812dd3b8b5b6/core/src/main/java/hudson/model/Result.java#L72

Reply:
https://github.com/jenkinsci/jenkins-design-language/blob/e01a2c2fdefcbda64318aa4422859d33e85b7a0d/docs/jdl-storybook/css/jenkins-design-language.css#L3215-L3232 i have now added all types that i have found in the design css file. Not shure what is the correct one ;(

But i think anyone can override the setting but get an idea how they can work with more than 2 variants